functions/nvm.fish

```diff
@@ -79,20 +79,28 @@ function nvm --argument-names cmd v --description "Node version manager"
                 echo -e "Fetching \x1b[4m$url\x1b[24m\x1b[7m"
 
                 if ! command curl --progress-bar --location $url \
-                        | command tar --extract --gzip --directory $nvm_data/$v 2>/dev/null
-                    command rm -rf $nvm_data/$v
+                        | command tar --extract --gzip --strip-components=1 --directory $nvm_data/$v 2>/dev/null
```

Comment:
IIRC `--strip-components=1` won't work on FreeBSD.

Reply:
I see... It'll have to converge here then